docs/validator/running-a-validator.md

```diff
@@ -0,0 +1,252 @@
+---
+id: running-a-validator
```

Comment:
Most of these steps require a bit more "meat" in the form of explanations and references to existing docs, and not just be the actual command that needs to be executed

The steps should explain what is happening and why

Reply:
a7c33cbb4756b8400485e7af8e44118724f7289c